plugins/include/amxmodx.inc

```diff
@@ -1828,7 +1828,7 @@ native set_user_flags(index, flags = -1, id = 0);
  * @param index     Client index, 0 to set flags of server
  * @param id        Flag set id, ranging from 0 to 31
  *
- * @noreturn
+ * @return          Bitflag sum of client's admin flags
```

Comment:
Maybe just `Bitsum` instead of `Bitflag sum`? Or `Bit vector`?

Reply:
Well all other amxmodx.inc natives use Bitflag sum, so for consistency its better to either keep it as it is or change them all. also whats wrong with bitflag ? https://en.wiktionary.org/wiki/bitflag

Reply:
@justgo97 hm, ok.